### PR TITLE
[minor] Pass options.compression verbatim to uws

### PR DESCRIPTION
**Pull request by alexhultman**
This allows to pass false, true, 0, 1, undefined like before but also 2

### CODE REVIEW

**Comment by lpinca** [comment]
For curiosity, what are the valid values and what is their meaning?

**Comment by lpinca** [comment]
Answer is in #654.

**Comment by alexhultman** [comment]
This is the new behavior of v0.14.8: https://github.com/uNetworking/uWebSockets-bindings/commit/b649c8a73eaab6484f95965bf6d8a86371562b17

Basically it takes 0, 1 or 17 so right now Primus does not allow the 17

File: transformers/uws/server.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ module.exports = function server() {
   }, this.primus.options.transport);
 
   const group = native.server.group.create(
-    opts.perMessageDeflate ? 1 : 0,
+    opts.perMessageDeflate,
```
Comment (lpinca) [inline transformers/uws/server.js]: How about using `opts.perMessageDeflate >>> 0` to make sure that an integer is passed to the native layer? Or is it already done there?

Comment (alexhultman) [reply transformers/uws/server.js]: The addon always receives an integer no matter if it makes sense or not from JS land

Comment (alexhultman) [reply transformers/uws/server.js]: Sure. If that is how all other transports work then go ahead

**Pull request closed without merging**